js/start.js

```diff
@@ -104,18 +105,26 @@ function fetchConfig() {
 const onboardingNeededDeferred = $.Deferred();
 let profileFetch;
 let settingsFetch;
+let ownFollowersFetch;
+let ownFollowingFetch;
 let onboardProfile = false;
 let onboardSettings = false;
 let profileFailed;
 let settingsFailed;
+let ownFollowersFailed;
+let ownFollowingFailed;
 
 function isOnboardingNeeded() {
   profileFetch = !profileFetch || profileFailed ?
     app.profile.fetch() : profileFetch;
   settingsFetch = !settingsFetch || settingsFailed ?
     app.settings.fetch() : settingsFetch;
+  ownFollowersFetch = !ownFollowersFetch || ownFollowersFailed ?
```

Comment:
My Bad!

I totally wrongly advised you on where to put this. This shouldn't be here, because obtaining the followers has absolutely nothing to do with determining if onboarding is needed.

Would you mind reverting this and breaking the following fetches into their own function? It could be named something "follow" specific or some generic "fetchStartupData". It should only be called if it's determined onboarding is not needed, maybe augment the 'onboardIfNeeded' function, e.g:

```
function onboardIfNeeded() {
  isOnboardingNeeded().done((onboardingNeeded) => {
    if (onboardingNeeded) {
      // let's go onboard
      onboard().done(() => onboardIfNeededDeferred.resolve());
    } else {
        fetchStartupData().done(() => onboardIfNeededDeferred.resolve());  // updated this line, rest is as-is from start.js      
    }
  });

  return onboardIfNeededDeferred.promise();
}
```

Really sorry for the misdirection.